docs/make.jl

```diff
@@ -7,6 +7,7 @@ DocMeta.setdocmeta!(Survey, :DocTestSetup, :(using Survey); recursive=true)
 makedocs(;
     modules=[Survey],
     authors="xKDR Forum",
+    # doctest = :fix, 
```

Comment:
I have intentionally left this comment. It's an imperfection in the package, but makes our life a lot simpler. @smishr @iuliadmtru

Reply:
Yes. Using this in make simply updates all the docstest outputs to new API.